AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -111,14 +120,70 @@ public final class UPIComponent: PaymentComponent,
     /// The  virtual payment address text input item.
     internal lazy var virtualPaymentAddressItem: FormTextInputItem = {
         let item = FormTextInputItem(style: configuration.style.textField)
-        item.title = Constants.virtualPaymentAddress
+        // TODO: Add localised string
+        item.title = "Enter UPI ID / VPA"
         item.validator = LengthValidator(minimumLength: 1)
         item.validationFailureMessage = localizedString(.UPIVpaValidationMessage, configuration.localizationParameters)
         item.identifier = ViewIdentifierBuilder.build(scopeInstance: self,
                                                       postfix: ViewIdentifier.virtualPaymentAddressInputItem)
         return item
     }()
 
+    /// The UPI app list item.
+    internal lazy var upiAppsList: [SelectableFormItem] = {
+        var upiAppslist = [SelectableFormItem]()
+        _ = upiPaymentMethod.apps.map { app -> SelectableFormItem in
```

Comment:
Is there a reason you don't use the result of the map`

If you don't want to use it you could also use:
```suggestion
        upiPaymentMethod.apps.forEach { app in
```

Reply:
It would maybe be also nice to encapsulate the creation of a `SelectableFormItem` from an app into a function so the lazy var content is smaller and easier to read

Reply:
eab7d68b538ab19f294c4b62fcffe59a625ae3d7